tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py

```diff
@@ -17,7 +17,7 @@
 # <snippet name="tests/integration/docusaurus/expectations/how_to_create_and_edit_an_expectationsuite_domain_knowledge.py get_data_context">
 import great_expectations as gx
 
-context = gx.data_context.FileDataContext.create(full_path_to_project_directory)
+context = gx.get_context(project_root_dir=full_path_to_project_directory)
```

Comment:
Does this new call also create a `FileDataContext`?

Reply:
yup! but we need the specific `get_context` to enable the factory work @joshua-stauffer did